src/SDKs/SqlManagement/Sql.Tests/VirtualNetworkRulesCrudScenarioTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Management.Resources;
+using Microsoft.Azure.Management.Sql;
+using Microsoft.Azure.Management.Sql.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace Sql.Tests
+{
+    public class VirtualNetworkRulesCrudScenarioTests
+    {
+        [Fact]
+        public void TestGetAndListVirtualNetworkRule()
+        {
+            string testPrefix = "virtualnetworkrulescrudtest-";
+            string suiteName = this.GetType().FullName;
+            SqlManagementTestUtilities.RunTestInNewV12Server(suiteName, "TestGetAndListVirtualNetworkRule", testPrefix, (resClient, sqlClient, resourceGroup, server) =>
+            {
+                // Create Firewall Rules
+                //
+                Dictionary<string, VirtualNetworkRule> rules = new Dictionary<string, VirtualNetworkRule>();
+
+                string vnetfirewallRuleName = SqlManagementTestUtilities.GenerateName(testPrefix);
+                VirtualNetworkRule rule = new VirtualNetworkRule()
+                {
+                    VirtualNetworkSubnetId = string.Format("/subscriptions/d513e2e9-97db-40f6-8d1a-ab3b340cc81a/resourceGroups/TestVnetSdk/providers/Microsoft.Network/virtualNetworks/vnetSdkTest/subnets/subnet1")
```

Comment:
Yes this is bad long-term, the problem is that the network RP's API to create these virtual networks is not fully working in production so it's not possible to automate right now. We'll make an issue assigned to @dnayantara to update this asap.